File: U_A_S_D/src/SORTIN_ALGORYTHMS/QUICK_SORT_183_02/QuickSortMain.java
package SORTIN_ALGORYTHMS.QUICK_SORT_183_02;

import java.util.Arrays;
import java.util.Scanner;

import static SORTIN_ALGORYTHMS.QUICK_SORT_183_02.QuickSortMethods.*;

public class QuickSortMain {

    public static void main(String[] args) {

        Scanner Input = new Scanner(System.in);

        int N = Input.nextInt(); // Number of elements
        //int N = Integer.parseInt(Input.nextLine()); // Same as above

        int[] ARR = new int[N]; // Define array of N elements

        // For cycle to fill up the array
        for (int i = 0; i < N; i++) {
            System.out.printf("Please, input the [%d] element : ", i);
            ARR[i] = Input.nextInt();
        }

        // BEFORE SORTING
        System.out.println("BEFORE SORTING : " + Arrays.toString(ARR));

        // Call QuickSort
        QuickSort(ARR,0,N-1);

        // AFTER SORTING
        System.out.println("AFTER SORTING : " + Arrays.toString(ARR));

    }

}

File: U_A_S_D/src/SORTIN_ALGORYTHMS/QUICK_SORT_183_02_KOTLIN/QuickSortMain.kt
package SORTIN_ALGORYTHMS.QUICK_SORT_183_02_KOTLIN

import java.util.*

object QuickSortMain {
    @JvmStatic
    fun main(args: Array<String>) {

        val input = Scanner(System.`in`)

        println("Input The Number Of Elements : ")
        val N = Integer.parseInt(input.nextLine())

        val arr = IntArray(N) // Define array of N elements

        // For cycle to fill up the array
        for (i in 0 until N) {
            println("Please, input the element [$i] element : ")
            arr[i] = input.nextInt()
        }

        // BEFORE SORTING
        println("BEFORE SORTING : " + arr.contentToString())

        // Call QuickSort
        quickSort(arr, 0, N - 1)

        // AFTER SORTING
        println("AFTER SORTING : " + arr.contentToString())
    }
}